gulpfile.js

```diff
@@ -2,10 +2,15 @@ const gulp = require("gulp");
 const util = require("gulp-util");
 const mocha = require("gulp-mocha");
 const config = require("./mocha.config");
+const xmlReporter = require("json-to-xml-reporter");
 
 gulp.task("test", () => {
     gulp.src("spec/*.spec.js")
         .pipe(mocha(Object.assign(config, {
             globals: [`env=${util.env.env}`]
-        })))
+        })));
+});
+
+gulp.task('creation:xmlReport', () => {
```

Comment:
mb rename to "report"?

Reply:
I thought about it, but I decided to name this task in that way because it's create only xml report and as for me it will be better to pick out what gulp task is doing exactly.